cluster/images/etcd/Makefile

```diff
@@ -83,7 +83,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version of the official build from https://github.com/etcd-io/etcd/releases.
-GOLANG_VERSION := 1.19.9
```

Comment:
are we even using this any more? can we remove `cluster/images/etcd`?

Reply:
Yes, :(

Reply:
where?

Reply:
This is the scripting and wrapping for the official etcd kubernetes image. Over the standard etcd image it introduces "support" for upgrades and downgrades. 

The code could be removed when etcd officially supports storage downgrades and upgrades in v3.6 (etcdutl migrate). 

However, when v3.6 will be released is unknown.